solutions/gold/csa-MountainTime.mdx

```diff
@@ -78,23 +77,19 @@ struct DSU {
 
 		if (grid[y] > grid[x]) {
 			// we have the answer for the smaller component
-			for (int &i : comp[x]) {
-				ans[i] = intermediate > grid[i] ? 0 : grid[i] - intermediate;
-			}
+			for (int &i : comp[x]) { ans[i] = grid[i] - intermediate; }
```

Comment:
no yeah now i'm a bit concerned if this assumes operations are performed in a certain order

Reply:
hmm well all of component y's elements will be greater than component x's - and intermediate (the element we just added which is the lowest) connects them.

i can add a comment if it's still unclear though ig

Reply:
comment would be nice

Reply:
okay sure